src/nationalarchives/components/nested-navigation/nested-navigation.stories.ts

```diff
@@ -237,34 +267,27 @@ ClickSelectAndKeyboardDeselect.args = {
 ClickSelectAndKeyboardDeselect.play = async ({ canvasElement }) => {
   const canvas = within(canvasElement);
 
-  await userEvent.click(canvas.getByText("Baking-powder Nov 1999.docx"));
-
-  await expect(
+  await userEvent.click(
     canvas.getByRole("treeitem", { name: "Baking-powder Nov 1999.docx" })
-  ).toBeChecked();
+  );
 
+  // await expect(
```

Comment:
Is this commented out code needed? If not it should be deleted.

Reply:
Removed.